### DISCUSSION TITLE
Visual refresh

### DISCUSSION THREAD

**Discussion by redudant-overflowing** [darkreader/darkreader#7494] [Ideas]
![Dark Reader (4nd revision)(1)](https://user-images.githubusercontent.com/93951203/143839574-ad64e21e-8a9d-4645-9fb1-cdb28e3de5d8.png)
![Dark Reader (3nd revision)](https://user-images.githubusercontent.com/93951203/143837819-a8e5c233-8ca0-4102-bf26-5c15e0a6aa61.png)
![Dark Reader (3nd revision)(1)](https://user-images.githubusercontent.com/93951203/143837826-c2ad539d-c7fd-479e-a841-a8fef46e46aa.png)

I made a [redesign concept in Figma](https://www.figma.com/file/9Q3wTBp3mzH7ogyFJ002kM/Dark-Reader-(4nd-revision)?node-id=0%3A1), but keeping the roots of the old design. This is what I wanted to achieve with this design:

- Appealing UI;
- Reduce clutter;
- Instead of the text of the focused tab being orange, which creates inconsistency, there is a little line at the bottom to keep the old identity;
- Creation of a settings page, because the current vertical ellipsis button is too tiny and hidden;
- Creating of an about page to put the GitHub, Twitter, Help and Privacy links, to reduce clutter;
- Better contrast for better accessibility;
- No flyouts with opacity, like the settings one, which has poor contrast;
- Little spacement between some items to reduce the clutter, but without using excessive padding;
- Version number in the about page for easier version checking;
- Removal of the white border. It looks ugly.

# Current UI (for comparison)
![Screenshot_20211128_182720-1](https://user-images.githubusercontent.com/93951203/143786685-3821b87c-c213-4d36-81fb-5136a85b8cb7.png)

**Comment by Gusted** [comment]
@alexanderby

**Comment by alexanderby** [comment]
Some ideas are cool, we will consider it when inventing a newer design.
